derive_client/data/rpc_endpoints.yaml

```diff
@@ -0,0 +1,59 @@
+ETH:
+  - https://eth.drpc.org
+  - https://eth.blockrazor.xyz
+  - https://rpc.mevblocker.io
+  - https://1rpc.io/eth
+  - https://eth.llamarpc.com
+  - https://ethereum-rpc.publicnode.com
+  - https://rpc.flashbots.net/
+  - https://eth.merkle.io/
+  - https://mainnet.gateway.tenderly.co
+
+OPTIMISM:
+  - https://optimism.drpc.org
+  - https://mainnet.optimism.io
+  - https://optimism-rpc.publicnode.com
+  - https://0xrpc.io/op
+  - https://1rpc.io/op
+  - https://op-pokt.nodies.app
+  - https://optimism.gateway.tenderly.co
+  - https://optimism.api.onfinality.io/public
+
+BASE:
+  # - https://base.drpc.org
+  - https://base.llamarpc.com
+  - https://mainnet.base.org
+  - https://base-rpc.publicnode.com
+  - https://0xrpc.io/base
+  - https://1rpc.io/base
+  - https://base.therpc.io
+  - https://base-pokt.nodies.app
+  - https://base.api.onfinality.io/public
```

Comment:
I left this one (`https://base.drpc.org`), since it's the odd one out hitting `TOO_MANY_REQUESTS` on first hit and will likely recover